### PR TITLE
Admin ui

### PR DESCRIPTION
**Pull request by arnaud-morvan**
Add web admin module.

* Admin main page with links to each hazard category.
* Hazard category form allowing to change general recommandation content and technical recommandations order.
* List,add and edit technical recommandations, add and delete associations with hazard categories.

Note that jquery-ui does not pass the jshint check, so I've put it outside of js folder.

### CODE REVIEW

**Comment by arnaud-morvan** [comment]
@pgiraud : could you review this please.

**Comment by pgiraud** [comment]
"Recommendation" is written with an "e" in English.

File: thinkhazard/tests/views/test_admin.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        resp = self.testapp.get('/admin/technical_rec/new', status=200)
+        form = resp.form
+        form['text'] = 'Bar'
+        form['associations[]'] = ['EQ - MED', 'EQ - LOW']
+        form.submit(status=302)
+
+    def test_technical_rec_edit(self):
+        resp = self.testapp.get('/admin/technical_rec/7', status=200)
+        form = resp.form
+        # here we should got ['EQ - HIG']
```
Comment (pgiraud) [inline thinkhazard/tests/views/test_admin.py]: What does this mean?

File: thinkhazard/views/admin.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+            'hazard_types': hazard_types,
+            'hazard_levels': hazard_levels,
+        }
+
+    if request.method == 'POST':
+        obj.text = request.POST.get('text')
+        if inspect(obj).transient:
+            DBSession.add(obj)
+
+        associations = request.POST.getall('associations[]')
```
Comment (pgiraud) [inline thinkhazard/views/admin.py]: Using '[]' is not required here.

File: thinkhazard/tests/views/test_admin.py (hunk 1 above)
Comment (arnaud-morvan) [inline thinkhazard/tests/views/test_admin.py]: ['EQ - HIG'] is the value that we get from the database, I note this, to be sure to delete and add some relations.

**Pull request merged**